android/sdl_android/src/main/java/com/smartdevicelink/transport/utl/SdlDeviceListener.java

```diff
@@ -174,9 +185,88 @@ public void handleMessage(@NonNull Message msg) {
                     break;
 
                 case com.smartdevicelink.transport.SdlRouterService.MESSAGE_READ:
+                    onPacketRead((SdlPacket) msg.obj);
                     break;
             }
         }
+
+        /**
+         * This method will start the RPC service so that we can retrieve vehicle info
+         */
+        public void sendStartService() {
+            SdlDeviceListener sdlListener = this.provider.get();
+            SdlPacket serviceProbe = SdlPacketFactory.createStartSession(SessionType.RPC, 0x00, (byte)0x01, (byte)0x00, false);
+            serviceProbe.putTag(ControlFrameTags.RPC.StartService.PROTOCOL_VERSION, MAX_PROTOCOL_VERSION.toString());
+            byte[] constructed = serviceProbe.constructPacket();
+            if (sdlListener.bluetoothTransport != null && sdlListener.bluetoothTransport.getState() == MultiplexBluetoothTransport.STATE_CONNECTED) {
+                sdlListener.bluetoothTransport.write(constructed, 0, constructed.length);
+            }
```

Comment:
We should likely handle the case where this does not send the packet. Otherwise the logic will be waiting indefinitely for an ACK that will never come.